docs/src/examples/components/Chat/index.tsx

```diff
@@ -1,11 +1,9 @@
 import React from 'react'
 import Types from './Types'
-import Variations from './Variations'
 
 const MessageExamples = () => (
```

Comment:
I know it's out of scope of PR, but we might want to rename to `ChatExamples` to avoid confusion

Reply:
Sure, I will restructure to example containing all different chat items. Will consider this as well.